libcaf_core/src/stream_manager.cpp

```diff
@@ -135,12 +138,12 @@ void stream_manager::stop(error reason) {
 
 void stream_manager::shutdown() {
   CAF_LOG_TRACE("");
-  // Mark as shutting down and reset other flags.
-  if (shutting_down())
```

Comment:
The header has a call to [`shutting_down()`](https://github.com/actor-framework/actor-framework/blob/d52371cff540c11d968b960a7643633371371902/libcaf_core/caf/stream_manager.hpp#L170) as well. I was wondering if that should test for `is_stopped_flag`. And if so, maybe `shutting_down()` should just check both flags?

Reply:
Good catch. Will fix.